src/lib.rs

```diff
@@ -625,20 +610,15 @@ impl Lockfile {
   ) -> Result<Lockfile, Box<LockfileError>> {
     fn load_content(
       content: &str,
-      next_version: bool,
     ) -> Result<LockfileContent, LockfileErrorReason> {
       let value: serde_json::Map<String, serde_json::Value> =
         serde_json::from_str(content)
           .map_err(LockfileErrorReason::ParseError)?;
       let version = value.get("version").and_then(|v| v.as_str());
       let value = match version {
-        Some("5") if next_version => value,
+        Some("5") => value,
         Some("4") => {
-          if next_version {
-            return Err(LockfileErrorReason::TransformNeeded);
-          } else {
-            value
-          }
+          return Err(LockfileErrorReason::TransformNeeded);
```

Comment:
Do we still need this `TransformNeeded`? I think this was worked around in the cli by moving loading the lockfile out of cli options?

Reply:
ah good point, removed it